frame/system/extensions/check_genesis.go

```diff
@@ -4,17 +4,19 @@ import (
 	"bytes"
 
 	sc "github.com/LimeChain/goscale"
-	"github.com/LimeChain/gosemble/constants/metadata"
 	"github.com/LimeChain/gosemble/frame/system"
 	primitives "github.com/LimeChain/gosemble/primitives/types"
 )
 
 type CheckGenesis struct {
-	module system.Module
+	module               system.Module
+	additionalSignedData sc.VaryingData
```

Comment:
Could we rename it to more accurately reflect the field’s purpose? For instance, `typesInfoAdditionalSignedData` or something along those lines?